domains/vaibhav-op.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "vaibhav-OP",
+           "email": "vaibhavyash28@gmail.com",
+           "discord": "414622975598395398"
+        },
+    
+        "record": {
+            "TXT": "vc-domain-verify=vaibhav-op.is-a.dev,d17d9c2f987342e656b7"
```

Comment:
TXT record alone do nothing, also vercel is not supported

Reply:
For the txt part i used your discord bot ro create the PR